src/components/button.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+type ButtonProps = React.ButtonHTMLAttributes<HTMLButtonElement> & {
+  intent?: 'default' | 'delete' | 'out_line';
+  size?: 'default' | 'small';
+};
+
+const CustomButton: React.FC<ButtonProps> = ({
+  intent = 'default',
+  size = 'default',
+  ...props
+}) => {
```

Comment:
P1: 컴포넌트에 `React.FC` 를 다 붙일 필요 없을 것 같습니다. 아래와 같이 변경해주세요.

```suggestion
const CustomButton = ({
  intent = 'default',
  size = 'default',
  ...props
}: ButtonProps) => {
```

Reply:
React.FC 제거